cartography/data/jobs/analysis/aws_lambda_ecr.json

```diff
@@ -3,12 +3,14 @@
     "statements": [
         {
             "__comment": "Create HAS_IMAGE realtionship from lambda functions to the associated ECR image",
-            "query":"MATCH (l:AWSLambda), (e:ECRImage)\nWHERE e.digest = 'sha256:' + l.codesha256\nMERGE (l)-[:HAS]->(e)",
-            "iterative": false
+            "query":"MATCH (l:AWSLambda) \n WITH COLLECT(l) as lmbda_list \n UNWIND lmbda_list as lmbda \n MATCH (e:ECRImage) \n WHERE e.digest = 'sha256:' + lmbda.codesha256 \n MERGE (lmbda)-[r:HAS]->(e) \n SET r.lastupdated = {UPDATE_TAG}",
+            "iterative": true,
```

Comment:
This has iterative=true but the query itself does not use the `{LIMIT_SIZE}` parameter so it won't work as intended. See example: https://github.com/lyft/cartography/blob/6d4be34ae0cea6ff3daa748aae374d7e26d5abb3/cartography/data/jobs/cleanup/aws_import_tags_cleanup.json#L4.

Reply:
Removed. I can't yet think of a good way to make this query iterative.